### PR TITLE
NEW: check changed state text fields on keyup, rather than just change

### PR DESCRIPTION
**Pull request by unclecheese**
This is a continuation of an old PR https://github.com/silverstripe/silverstripe-framework/pull/3592 but against the newer branch.

### CODE REVIEW

**Comment by tractorcow** [comment]
I think this fix will add a lot of value. I'm keen on testing it out, but it looks like tests are broken. Do you want to have a go at fixing them?

File: admin/javascript/jquery-changetracker/lib/jquery.changetracker.js [hunk 1]
```diff
@@ -88,9 +89,31 @@
...
+
+			fields.filter(omittedSelectors).on('click.changetracker', onchange);
+			fields.not(omittedSelectors).on('change.changetracker', onchange);
+			
+			// Handle omitted selectors
 			fields.each(function() {
+				// For selectors with custom change events
+				for(var selector in options.customChangeEvents) {
+					var eventName;
+					if(options.customChangeEvents.hasOwnProperty(selector)) {
```
Comment (tractorcow) [inline admin/javascript/jquery-changetracker/lib/jquery.changetracker.js]: How could this be false? Isn't it a property by virtue of the loop on options.customChangeEvents?

**Comment by tractorcow** [comment]
I'm not smart enough to see any issues in the code itself. :) Good job @unclecheese .

**Comment by unclecheese** [comment]
This whole PR is bunk now, as all of this has moved to `silverstripe-admin`. Will start a new one and close this one off.

**Pull request closed without merging**